### PR TITLE
New Unit Select

### PR DESCRIPTION
**Pull request by Voz-Duh**
This looks like good
Big units(blocks):
image

### CODE REVIEW

File: core/src/mindustry/graphics/OverlayRenderer.java [hunk 1]
```diff
@@ -88,8 +88,11 @@ public void drawTop(){
                 Draw.rect(select.type.icon(Cicon.full), select.x(), select.y(), select.rotation() - 90);
             }
 
-            Lines.stroke(unitFade);
-            Lines.square(select.x, select.y, select.hitSize() * 1.5f, Time.time() * 2f);
+            for(int i = 0; i < 4; i++){
+                float rot = i * 90f + 45f + (-Time.time() / 3f) % 360f;
+                float length = select.hitSize() * 1.5f + (unitFade * 2.5f);
+                Draw.rect(Core.atlas.find("select-arrow"), select.x + Angles.trnsx(rot, length), select.y + Angles.trnsy(rot, length), length / 1.9f, length / 1.9f, rot - 180f);
```
Comment (summetdev) [inline core/src/mindustry/graphics/OverlayRenderer.java:94]: Maybe cache this `select-arrow` region?

Comment (Voz-Duh) [reply core/src/mindustry/graphics/OverlayRenderer.java:95]: Okay, I go do it

**Comment by DarkBrick** [comment]
Arrow sprites seem a bit off but otherwise great suggestion

**Comment by Voz-Duh** [comment]
?

**Comment by Anuken** [comment]
The arrows should be a straight 45 degrees, instead of having several slopes.

**Comment by Voz-Duh** [comment]
image

**Comment by Voz-Duh** [comment]
@Anuken, I maked all of your demands

**Comment by Voz-Duh** [comment]
It's been 4 days, @Anuken didn't notice that I did everything he told me _/-/_

**Comment by Anuken** [comment]
image

This is problematic - scaling up an image like this leads to jagged edges, which can be very obvious on bigger units and don't look very good.

**Comment by Catchears** [comment]
> It's been 4 days, (@)Anuken didn't notice that I did everything he told me /-/

You realise that Anuke does have other things to do? I don't think mentioning him will help. He'll comment / close / merge / request changes when he has time.

**Comment by Voz-Duh** [comment]
> image
> 
> This is problematic - scaling up an image like this leads to jagged edges, which can be very obvious on bigger units and don't look very good.

I created a new special big sprite for it 
There are no such problems with him and everything looks beautiful

**Comment by Voz-Duh** [comment]
@Anuken, this for example:
image

**Pull request merged**